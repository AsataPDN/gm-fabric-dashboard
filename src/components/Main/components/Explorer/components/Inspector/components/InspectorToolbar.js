import styled from "styled-components";

import { COLOR_CONTENT_BACKGROUND } from "style/styleVariables";
import { edgeColor, spacingScale } from "style/styleFunctions";

const InspectorToolbar = styled.div`
  align-items: stretch;
  box-shadow: 0 1px 0 0 ${edgeColor(COLOR_CONTENT_BACKGROUND).string()};
  display: flex;
  flex: 0 0 ${spacingScale(7)};
  padding: ${spacingScale(1)};
`;

export default InspectorToolbar;
