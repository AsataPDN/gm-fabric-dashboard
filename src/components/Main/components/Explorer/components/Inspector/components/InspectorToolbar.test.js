import React from "react";
import { shallow } from "enzyme";
import InspectorToolbar from "./InspectorToolbar";

describe("InspectorToolbar", () => {
  it("matches snapshot", () => {
    const aInspectorToolbar = shallow(<InspectorToolbar />);
    expect(aInspectorToolbar).toMatchSnapshot();
  });
});
