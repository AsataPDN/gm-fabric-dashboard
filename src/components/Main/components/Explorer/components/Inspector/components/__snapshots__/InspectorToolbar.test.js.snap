// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InspectorToolbar matches snapshot 1`] = `
.c0 {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  box-shadow: 0 1px 0 0 hsl(0,0%,92%);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 0 56px;
  -ms-flex: 0 0 56px;
  flex: 0 0 56px;
  padding: 8px;
}

<div
  className="c0"
/>
`;
